test/composites/alert/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const assert = require('chai').assert;
+const Classlist = require('classlist');
```

Comment:
Why are we bothering to use a module to polyfill `el.classList`? Isn't DQPL's support matrix exclusively paltforms that support `el.classList`?